pyat/at/lattice/transformation.py

```diff
@@ -0,0 +1,280 @@
+import numpy
+#import re
+from typing import Union, Tuple, List, Type
```

Comment:
Adding this line on top will ensure compatibility with old python version (3.7 for us):
```python
from __future__ import annotations
```
Also, `Union, Tuple, List, Type` are not used, you can remove this line.